kernel/src/codec_manager.cpp

```diff
@@ -59,4 +73,31 @@ status_t getCodec(std::string name, CodecInfo &codec_info)
     }
 
     return 1;
+}
+
+//fun what search files whith need extension
+std::list <std::string> recursiveDLLSearch(const std::string& search_path, const std::string& extension) {
+
+	std::list <std::string> list_of_path_to_dll; // create local list? what contained pathes
+
+	try {

```

Comment:
Why you used methods that throws exceptions?

Reply:
Because I couldn`t find other methods